episodes/wrap-up3.md

```diff
@@ -20,10 +20,66 @@ After completing this episode, participants should be able to...
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+We have reached the end of this training.
+
+## Looking Back
+
+Throughout this training we promoted a collaborative approach to lesson development and maintenance (as many other initiatives in The Carpentries community are conducted). Collaboration helps bring different skills and perspectives to a project and share the load of the work required to bring the project successfully past the finishing line. For lesson projects, this shared effort also extends to maintaining and updating the lesson after the initial development phase.
+
+::::::::::::: challenge
+
+# Exercise: What steps can you take to be a good collaborator? (20 minutes)
```

Comment:
```suggestion
## Exercise: What steps can you take to be a good collaborator? (20 minutes)
```
Don't think this should be a level 1 header, my understanding it is best practice is that is only used once for the top level of the page.

Reply:
Not sure - I copied the exercise block from another exercise and it had the level 1 heading. @tobyhodges  will know better - will check with him.

Reply:
Fixed via 35e6912a.